copter/source/docs/traditional-helicopter-autotune.rst

```diff
@@ -0,0 +1,199 @@
+
+===============================
+AutoTune
+===============================
+The AutoTune for tradheli is completely different from multicopter AutoTune.  It can tune any combination of feedforward (ATC_RAT_XXX_FF), the rate gains (ATC_RAT_XXX_P and ATC_RAT_XXX_D), or angle P gain (ATC_ANG_XXX_P).  The tuning for rate gains begins with finding the maximum allowable value for the rate gains and then tunes them.  Knowing the maximum value enables the AutoTune feature to keep from creating an instability.
+
+Before you start AutoTune, you must:
+
+#. Reduce the vibrations in the control signals as low as possible
+#. On a calm day, the heli will maintain a near driftless hover (< 1 m/s)
+
+This will greatly improve your chances at a successful AutoTune.
+
+Parameter Descriptions
+======================
+:ref:`Tune Sequence Bitmask<AUTOTUNE_SEQ>`
+------------------------------------------
+
+User can specify the tuning desired.  Individual gain tuning or combination of tuning several gains can be specified using the bitmask.  Rate P and Rate D are conducted together and always preceeded with determining max gains allowable.
+
++-----------------------------+------------------------------------+
+| Bits                        | Values                             |
++=============================+====================================+
+| +-------+-----------------+ | +---------+----------------------+ |
+| | Bits  | Tune Type       | | | Values  | Tune Type            | |
+| +=======+=================+ | +=========+======================+ |
+| | 0     | Feedforward     | | | 1       | Feedforward          | |
+| +-------+-----------------+ | +---------+----------------------+ |
+| | 1     | Rate D & Rate P | | | 2       | Rate D* & Rate P     | |
+| +-------+-----------------+ | +---------+----------------------+ |
+| | 2     | Angle P         | | | 3       | Feedforward, Rate D*,| |
+| +-------+-----------------+ | |         | & Rate P             | |
+| | 3     | Max Gain        | | +---------+----------------------+ |
+| +-------+-----------------+ | | 4       | Angle P              | |
+|                             | +---------+----------------------+ |
+|                             | | 5       | Feedforward & Angle P| |
+|                             | +---------+----------------------+ |
+|                             | | 6       | Rate D*, Rate P,     | |
+|                             | |         | & Angle P            | |
+|                             | +---------+----------------------+ |
+|                             | | 7       | Feedforward, Rate D*,| |
+|                             | |         | Rate P, & Angle P    | |
+|                             | +---------+----------------------+ |
+|                             | | 8       | Max Gain             | |
+|                             | +---------+----------------------+ |
++-----------------------------+------------------------------------+
+
```

Comment:
I think that the right hand table should refer to the actual param label rather than a descriptive name, ie ATC_RAT_xxx_P

Reply:
I can work on this after merge if you like